packages/babel-traverse/src/path/introspection.js

```diff
@@ -158,7 +158,16 @@ export function isStatementOrBlock() {
  */
 
 export function referencesImport(moduleSource, importName) {
-  if (!this.isReferencedIdentifier()) return false;
+  if (!this.isReferencedIdentifier()) {
+    if (
+      this.isMemberExpression() &&
+      !this.node.computed &&
+      this.node.property.name === importName
```

Comment:
Since `export { foo as "😄" }` is valid now, we should also check if `this.node.property` is a `StringLiteral` and if so, compared with its `value`.

Reply:
IIUC you're saying "we should extend this to support the simplest computed property case, because exports with certain names can only be accessed that way". Will implement